event_location_manager/models/event_event.py

```diff
@@ -0,0 +1,130 @@
+# -*- coding: utf-8 -*-
+# @authors: Alexander Ezquevo <alexander@acysos.com>
+# Copyright (C) 2016  Acysos S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api, _
+from openerp.exceptions import Warning
+from datetime import datetime, timedelta
+
+
+DFORMAT = "%Y-%m-%d %H:%M:%S"
+
+
+class EventTrackLocation(models.Model):
+    _inherit = 'event.track.location'
+
+    def _get_company(self):
+        return self.env.user.company_id
+
+    company_id = fields.Many2one(string='Company', comodel_name='res.company',
+                                 default=_get_company)
+    capacity = fields.Integer(string='Capacity', required=True)
+    reservation_days = fields.One2many(
+        string='Reservation days',
+        comodel_name='event.track.location.reservation',
+        inverse_name='et_location_id')
+
+
+class EventTrackLocationReservation(models.Model):
+    _name = 'event.track.location.reservation'
+
+    @api.depends('day', 'duration')
+    @api.multi
+    def _get_end_date(self):
+        for res in self:
+            if res.day:
+                date = datetime.strptime(res.day, DFORMAT)
+                res.end_date = date + timedelta(hours=res.duration)
+    et_location_id = fields.Many2one(string='Location',
+                                     comodel_name='event.track.location',
+                                     requiered=True)
+    day = fields.Datetime(string='start date', requiered=True)
+    duration = fields.Float(string='duration')
+    end_date = fields.Datetime(string='End date', compute=_get_end_date,
+                               store=True)
+    track_id = fields.Many2one(string='Session', comodel_name='event.track')
+
+    def check_avairavility(self, location, day, ide, duration):
```

Comment:
Other mistype, this time it does not affect functionality.... it should be **availability**